src/home/homed-manager-bus.c

```diff
@@ -482,6 +494,16 @@ static int method_create_home(
         if (r < 0)
                 return r;
 
+        if (extended) {
+                r = bus_message_read_blobs(message, &blobs, error);
+                if (r < 0)
+                        return r;
+
+                r = sd_bus_message_read(message, "t", &flags);
+                if (r < 0)
+                        return r;
```

Comment:
please refuse unrecognized flags early with a not supported error, so that we can later add more flags and clients have a nice way to detect whether they are supported: just use them and if they get an error back they aren't